src/composables/currencies.ts

```diff
@@ -0,0 +1,89 @@
+import { computed, ref } from '@vue/composition-api';
+import type { CurrencyCode } from '../types';
+import {
+  CURRENCIES,
+  CURRENCIES_URL,
+  fetchJson,
+  getLocalStorageItem,
+  handleUnknownError,
+  setLocalStorageItem,
+} from '../popup/utils';
+import { createPollingBasedOnMountedComponents } from './composablesHelpers';
+
+const POLLING_INTERVAL = 3600000;
+const LOCAL_STORAGE_CURRENCY_KEY = 'currency';
+const APP_CURRENCY_CODES = CURRENCIES.map(({ code }) => code).join(',');
+
+const currencyRates = ref<Record<string, number>>({});
+const currentCurrencyCode = ref<CurrencyCode>(getLocalStorageItem<CurrencyCode>([LOCAL_STORAGE_CURRENCY_KEY]) || 'usd');
+
+const initPollingWatcher = createPollingBasedOnMountedComponents();
+
+export function useCurrencies() {
+  const minTipAmount = computed(() => 0.01 / currencyRates.value.usd);
```

Comment:
wouldn't the initial value for `currencyRates` be 0?

Reply:
![image](https://user-images.githubusercontent.com/8217670/222430623-0c01ebe9-4623-48da-9463-896925240e4f.png)
This is how it looked before. I just moved the code from one place to another. I'll try to address your concerns on the chat.

Reply:
I checked few things and the rate for euro is `0.070233`. So `0.01 / 0.070233 = 0,1428` - I think it's OK.

Reply:
```suggestion
  const minTipAmount = computed(() => currencyRates.value.usd && (0.01 / currencyRates.value.usd));
```
🤷‍♀️  or to ensure that you have some initial ae/usd price

Reply:
Ups, my mistake - I misunderstood your question. `currencyRates.value.usd` is undefined in the initial state, so `0.01 / undefined = NaN`. It's not causing error, but still is a mistake. I added a fallback to `1` in case of falsy value.